.github/workflows/humble-binary-build.yml

```diff
@@ -7,9 +7,19 @@ on:
   pull_request:
     branches:
       - humble
+    paths:
+      - '**.hpp'
+      - '**.h'
+      - '**.cpp'
+      - '.github/workflows/humble-binary-build.yml'
```

Comment:
It probably also makes sense to run on changes to 

- package.xml
- CMakeLists.txt
- repo files

That comment would be relevant for all build jobs.

Reply:
you are right, I'll check this